src/librawspeed/metadata/ColorFilterArray.h

```diff
@@ -27,16 +27,15 @@
 namespace RawSpeed {
 
 enum CFAColor {
-  CFA_COLOR_MIN = 0,
+  // see also DngDecoder
   CFA_RED = 0,
   CFA_GREEN = 1,
   CFA_BLUE = 2,
-  CFA_GREEN2 = 3,
+  CFA_INVALID = 3, // keep the unit tests happy
```

Comment:
> But you expressed your explicit worries about changing anything in that regard, so I just fixed #34 here.

Specifically because up until now it seems you did not understand *why* that GREEN2 was needed there for "constructor taking a dcraw_filter".